scripts/GameCorner.asm

```diff
@@ -48,31 +48,33 @@ GameCorner_ScriptPointers:
 	dw CeladonGameCornerScript2
 
 CeladonGameCornerScript0:
+	; Idle
 	ret
 
 CeladonGameCornerScript1:
+	; Rocket battle
 	ld a, [wIsInBattle]
 	cp $ff
-	jp z, CeladonGameCornerScript_48c07
+	jp z, CeladonGameCornerScript1_ReenterMapAfterPlayerLoss
 	ld a, $f0
 	ld [wJoyIgnore], a
-	ld a, $d
+	ld a, 13 ; Our hideout might be discovered! I better tell BOSS!
```

Comment:
Can this be left as `$d` for now? It would probably be better long-term to have defined constants for each text ID, rather than commenting every use/reference to one.